tiny_dnn/activations/activation_layer.h

```diff
@@ -101,6 +101,11 @@ class activation_layer : public layer {
                                    vec_t &dx,
                                    const vec_t &dy) = 0;
 
+  /**
+   * Target value range for learning.
+   */
+  virtual std::pair<float_t, float_t> scale() const = 0;
```

Comment:
yeah, but most of them returns same values. Why not just override the ones that are really needed